src/components/views/rooms/RoomHeader.js

```diff
@@ -225,12 +226,17 @@ module.exports = React.createClass({
                 roomName = this.props.room.name;
             }
 
+            const innerName =
+                <EmojiText element="div"
+                           className={ "mx_RoomHeader_nametext " + (settingsHint ? "mx_RoomHeader_settingsHint" : "") }
+                           title={ roomName }>{roomName}</EmojiText>;
+
+            if (this.props.onSettingsClick) {
```

Comment:
I'm pretty sure it's ok to pass `undefined` to `onClick`, so this is a bit redundant.